package.json

```diff
@@ -7,23 +7,36 @@
     "format:check": "prettier --check .",
     "format:write": "prettier --write .",
     "format:check:package": "prettier --check \"packages/${PACKAGE}/**/*\"",
+    "postinstall": "npm run lerna bootstrap",
     "lerna": "lerna",
     "lint": "lerna run lint",
-    "test": "jest"
+    "test": "lerna run test"
   },
   "devDependencies": {
-    "@types/jest": "^25.2.1",
+    "@angular/cli": "^8.2.0",
+    "@angular/common": "^8.2.0",
+    "@angular/compiler": "^8.2.0",
+    "@angular/compiler-cli": "^8.2.0",
+    "@angular/core": "^8.2.0",
+    "@angular/platform-browser": "^8.2.0",
+    "@angular/platform-browser-dynamic": "^8.2.0",
+    "@types/mocha": "^8.0.3",
+    "jest-preset-angular": "^8.3.2",
+    "zone.js": "~0.10.1",
+    "@angular-devkit/build-angular": "~0.802.0",
```

Comment:
After all run this command in the same dir where this file exists
`npx sort-package-json "package.json"`